packages/adapter-statsig/src/index.ts

```diff
@@ -100,7 +100,8 @@ export function createStatsigAdapter(options: {
     return user != null && typeof user === 'object';
   };
 
-  const syncHandler = createSyncingHandler();
+  const minSyncDelayMs = options.edgeConfig ? 20 : 5_000;
```

Comment:
Let's see if we can go with the solution where we identify requests and only sync at-most-once-per-request rather than time based.